ZIGSIMPlus/Entities/ModalTexts.swift

```diff
@@ -8,22 +8,23 @@
 
 import Foundation
 
-let modalTexts:[Command: String] = [
-    .acceleration : "test0\naccelerationです。",
-    .gravity : "test1\ngravityです。",
-    .gyro : "test2\ngyroです。",
-    .quaternion : "test3\nquaternionです。",
-    .compass : "test4\ncompassです。",
-    .pressure: "test5\npressureです。",
-    .gps : "test6\ngpsです。",
-    .touch : "test7\ntouchです。",
-    .beacon : "test8\nbeaconです。",
-    .proximity : "test9\nproximityです。",
-    .micLevel : "test10\nmicLevelです。",
-    .remoteControl : "test11\nremoteControlです。",
-    .ndi : "test12\nndiです。",
-    .nfc : "test13\nnfcです。",
-    .arkit : "test14\narkitです。",
-    .battery : "test16\nbatteryです。",
-    .applePencil : "test17\napplePencilです。"
+// Title and Body for help modal
+let modalTexts:[Command: (String, String)] = [
```

Comment:
How about naming tuple, like
`(title: String, bodyText: String)` ?

Reply:
Whoa, I didn't know that feature.
Fixed in 5dedec7 🙏